pkg/project/registry/proxy/proxy_test.go

```diff
@@ -70,24 +69,10 @@ func TestCreateProjectBadObject(t *testing.T) {
 	}
 }
 
-func TestCreateProjectMissingID(t *testing.T) {
```

Comment:
Ok.  Then you no long have a test that ensures there's a validation call in the REST.Create flow.  Looks like illegal display name would do it.